lambda/index.js

```diff
@@ -0,0 +1,23 @@
+const AWS = require('aws-sdk')
+const forge = require('node-forge')
+
+const rsa = forge.pki.rsa
+
+const secretsmanager = new AWS.SecretsManager()
+
+exports.handler = async (event, context) => {
+  try {
+    const keypair = rsa.generateKeyPair({ bits: 2048, e: 0x10001 })
```

Comment:
make the bits configurable? 4096 bit keys are not supported everywhere (fuck you, AWS) but are supported in most things.